src/box/txn_limbo.c

```diff
@@ -59,6 +59,7 @@ txn_limbo_create(struct txn_limbo *limbo)
 	limbo->frozen_reasons = 0;
 	limbo->is_frozen_until_promotion = true;
 	limbo->do_validate = false;
+	limbo->confirm_lag = 0;
```

Comment:
Nit: probably the same logic here. Confirm lag is 0, when the last entry was immediately confirmed. Initial value should be probably -1, nil should be pushed to lua table.

We have to discuss the default values for `age` and `confirm_lag` together. @Gerold103 @CuriousGeorgiy what do you think?

Reply:
> Confirm lag is 0, when the last entry was immediately confirmed.

I don't think it will ever be exactly 0.

Overall, I think there is no better default value, than 0. It exists only when the server never did any synchronous replication. An alternative would be to not push the `confirm_lag` field, if we see it is exactly 0.

Reply:
Same as for the other comment - I think value 0 is fine. Speaking of "immediately confirmed" - it isn't really possible. Confirm would have to happen in the matter of < microsecond which isn't possible. But regardless of that, having a negative lag looks inconvenient - it is not a valid time duration value. People might not expect that.